package.json

```diff
@@ -58,6 +58,7 @@
   "dependencies": {
     "bin-pack": "^1.0.2",
     "d3": "^7.6.1",
+    "d3-hierarchy": "^3.1.2",
```

Comment:
Rename commit `update dependencies` to `Add d3-hierarchy dependency`

Reply:
Now, also removed bin-pack and interactjs dependencies